index.html

```diff
@@ -1,16 +1,105 @@
-<!DOCTYPE html>
-<html lang="en">
+        <!DOCTYPE html>
+        <html lang="en">
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+        <head>
+            <meta charset="UTF-8">
+            <link rel="stylesheet" href="style.css">
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Cake website</title>
+            <meta http-equiv="X-UA-Compatible" content="ie=edge">
+            <title>Responsive Cake webpage</title>
+            
+        
+            <!-- Add a link to your css file here -->
+        </head>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+        <body>
+        <main>
+        <!--navigation and logo-->
+        <nav class="upper-navigation"> 
+            <header><center>Welcome to Sweet cakes!</center></header>
```

Comment:
`<center>` is not a valid html element - have a quick read on how to use the [header element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)